packages/application-components/src/components/dialogs/form-dialog/form-dialog.visualroute.js

```diff
@@ -103,6 +103,8 @@ const FormDialogOverflowExample = props => (
               onChange={formikProps.handleChange}
               onBlur={formikProps.handleBlur}
               onFocus={formikProps.handleFocus}
+              menuPortalTarget={document.body}
```

Comment:
just remove this test until we can figure out what's going on with Percy and positioning items with JavaScript 😄

Reply:
Yeah, I guess we can remove it for now. I'll clean things up.

Reply:
or you can do it in another PR 🤷‍♂